src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -335,35 +344,39 @@ def sync_deploy_root_with_stage(
         role: str,
         prune: bool,
         recursive: bool,
+        stage_fqn: str,
         local_paths_to_sync: List[Path] | None = None,
     ) -> DiffResult:
         """
         Ensures that the files on our remote stage match the artifacts we have in
         the local filesystem.
 
         Args:
+            bundle_map: the artifact mapping computed during the `bundle` step. Required when local_paths_to_sync is
```

Comment:
that's my bad, thanks for fixing it!

Reply:
It's always required now BTW, so we can remove the 2nd part of the sentence

Reply:
updated